t/e2e/whisper/whisper_test.go

```diff
@@ -109,7 +110,15 @@ func (s *WhisperTestSuite) TestSelectAccount() {
 	// select another account, make sure that previous account is wiped out from Whisper cache
 	s.False(whisperService.HasKeyPair(pubKey), "identity already present in whisper")
 	s.NoError(s.Backend.SelectAccount(address, TestConfig.Account1.Password))
-	s.True(whisperService.HasKeyPair(pubKey), "identity not injected into whisper")
+	// get the chat key of the selected account
```

Comment:
I would write it as a separate test `TestWalletAndChatPubKeyAreTheSame` and this test can be unchanged. 

Later, when we make the final change, this test will continue to pass and only `TestWalletAndChatPubKeyAreTheSame` will fail and will be replaced with a one verifying that the keys are different.

Reply:
@adambabik ok I can create a new test for this, good idea!